### ISSUE TITLE
Parsing of lyr format fails when the lyr file contains color code

### ISSUE THREAD

**Issue by lindjoha** [equinor/ecl2df#306]
The lyr format can contain color code:
https://resinsight.org/3d-main-window/formations/#formation-names-description-files-_lyr_

But the function parse_zonemapfile in the common module fails if color is given.

**Comment by berland** [comment]
Refactor `common.parse_zonemapfile` to be able to return more information pr `k` layer, and then include the colour attribute. `grid.merge_zones()` does not need to add the colour attribute, but must be updated to follow the change in `common.parse_zonemapfile`.

**Comment by lindjoha** [comment]
Is it better to have the output format like this?
```
{
    "zone_1":{
            "from_layer":1,
            "to_layer":10,
            "color":"#fsdfasd"
        },
    "zone_2":{
            "from_layer":10,
            "to_layer":12,
            "color":"#fsdfasd"
        }
}
```

**Comment by berland** [comment]
That could work, would imply moving the layer unrolling functionality into the function `merge_zones`.

**Comment by lindjoha** [comment]
yes, also in webviz i would have to do that.

**Comment by berland** [comment]
Fixed in #307
